src/helpers.ts

```diff
@@ -1,11 +1,16 @@
 import { BigNumber } from "@ethersproject/bignumber";
 
-import { Integer, SdkError, Usdc } from "./types";
+import { Address, Integer, SdkError, Usdc } from "./types";
 
 export const ZeroAddress = "0x0000000000000000000000000000000000000000";
 export const EthAddress = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```

Comment:
Yeah agree. The plan is to use the `ZeroAddress` representing the native token of each chain. Maybe we can use an alias for it to be more clear. Maybe good for a refactor later since it will probably need a change in many places where its used, and dont want to add more to this one as it will be harder to review.